syncd/VendorSai.cpp

```diff
@@ -663,7 +663,11 @@ sai_status_t VendorSai::bulkCreate(
 
         default:
             SWSS_LOG_ERROR("not implemented, FIXME");
-            return SAI_STATUS_FAILURE;
+            return SAI_STATUS_NOT_SUPPORTED;
```

Comment:
inconsistent return code, error message says not implemented, and return code not supported
you can also extend error message by %s and sai_serialize_object_type(object_type).c_str()